src/Microsoft.AspNetCore.Server.Kestrel/Internal/Infrastructure/HttpUtilities.cs

```diff
@@ -15,7 +15,13 @@ public static class HttpUtilities
         public const string Http10Version = "HTTP/1.0";
         public const string Http11Version = "HTTP/1.1";
 
+        public const string HttpUriScheme = "http://";
+        public const string HttpSecureUriScheme = "https://";
```

Comment:
I was intentionally trying to emphasize the "S" of https by expanding it to Secure. It's easy to miss that extra character when reviewing/writing code.